Fix coverage GitHub action error

```
Run ArtiomTr/jest-coverage-report-action@v2
[9](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:9)
Run $GITHUB_ACTION_PATH/run.sh
[24](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:24)
Running using node v16.14.0
[25](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:25)
Begin initialization stage...
[26](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:26)
Initialization stage ended
[27](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:27)
Begin stages.parsethreshold...
[28](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:28)
stages.parseThreshold ended
[29](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:29)
Begin head coverage collection...
[30](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:30)
Begin installing dependencies...
[31](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:31)
/usr/local/bin/npm install
[32](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:32)

[33](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:33)
added 836 packages, and audited 888 packages in 18s
[34](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:34)

[35](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:35)
145 packages are looking for funding
[36](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:36)
  run `npm fund` for details
[37](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:37)

[38](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:38)
found 0 vulnerabilities
[39](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:39)
Installing dependencies ended
[40](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:40)
Begin running tests...
[41](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:41)
/usr/local/bin/npx jest --ci --json --coverage --testLocationInResults --outputFile=report.json
[42](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:42)
PASS tests/better-switch-errors.test.ts
[43](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:43)
PASS tests/better-switch.test.ts
[44](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:44)
Jest: Coverage data for al was not found.
[45](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:45)

[46](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:46)
Test Suites: 2 passed, 2 total
[47](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:47)
Tests:       11 passed, 11 total
[48](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:48)
Snapshots:   0 total
[49](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:49)
Time:        2.799 s
[50](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:50)
Ran all test suites.
[51](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:51)
Test results written to: report.json
[52](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:52)
Running tests failed
[53](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:53)
Error: Error: The process '/usr/local/bin/npx' failed with exit code 1
[54](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:54)
Error: The process '/usr/local/bin/npx' failed with exit code 1
[55](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:55)
    at jp._setResult (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:17263)
[56](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:56)
    at jp.CheckComplete (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:16853)
[57](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:57)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:1[58](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:58)46)
58
    at ChildProcess.emit (node:events:520:28)
[59](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:59)
    at maybeClose (node:internal/child_process:1092:16)
[60](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:60)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
[61](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:61)
Running tests ended
[62](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:62)
Begin collecting coverage...
[63](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:63)
Loading code coverage from file: report.json
[64](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:64)
Collecting coverage ended
[65](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:65)
Begin parsing coverage...
[66](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:66)
Parsing coverage ended
[67](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:67)
Head coverage collection ended
[68](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:68)
Begin switching to base branch...
[69](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:69)
/usr/bin/git fetch --all --depth=1
[70](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:70)
Fetching origin
[71](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:71)
From https://github.com/mward-sudo/better-switch
[72](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:72)
   9cd0ce8..fbcfcad  dev        -> origin/dev
[73](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:73)
/usr/bin/git checkout -f dev
[74](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:74)
Previous HEAD position was 8061f8e Merge 041be60f6ff763deea29ccc48fa336176c570180 into 9cd0ce8e605cee060d5b3931f99ec7f5af893e8d
[75](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:75)
Switched to a new branch 'dev'
[76](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:76)
branch 'dev' set up to track 'origin/dev'.
[77](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:77)
Switching to base branch ended
[78](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:78)
Begin base coverage collection...
[79](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:79)
Begin installing dependencies...
[80](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:80)
/usr/local/bin/npm install
[81](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:81)

[82](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:82)
added [83](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:83)6 packages, and audited 888 packages in 7s
83

[84](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:84)
145 packages are looking for funding
[85](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:85)
  run `npm fund` for details
[86](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:86)

[87](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:87)
found 0 vulnerabilities
[88](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:88)
Installing dependencies ended
[89](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:89)
Begin running tests...
[90](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:90)
/usr/local/bin/npx jest --ci --json --coverage --testLocationInResults --outputFile=report.json
[91](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:91)
PASS tests/better-switch-errors.test.ts
[92](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:92)
PASS tests/better-switch.test.ts
[93](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:93)
Jest: Coverage data for al was not found.
[94](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:94)

[95](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:95)
Test Suites: 2 passed, 2 total
[96](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:96)
Tests:       11 passed, 11 total
[97](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:97)
Snapshots:   0 total
[98](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:98)
Time:        1.912 s, estimated 3 s
[99](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:99)
Ran all test suites.
[100](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:100)
Test results written to: report.json
[101](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:101)
Running tests failed
[102](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:102)
Error: Error: The process '/usr/local/bin/npx' failed with exit code 1
[103](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:103)
Error: The process '/usr/local/bin/npx' failed with exit code 1
[104](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:104)
    at jp._setResult (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:17263)
[105](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:105)
    at jp.CheckComplete (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:16853)
[106](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:106)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/ArtiomTr/jest-coverage-report-action/v2/dist/index.js:10:15846)
[107](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:107)
    at ChildProcess.emit (node:events:520:28)
[108](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:108)
    at maybeClose (node:internal/child_process:[109](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:109)2:16)
109
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
[110](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:110)
Running tests ended
[111](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:111)
Begin collecting coverage...
[112](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:112)
Loading code coverage from file: report.json
[113](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:113)
Collecting coverage ended
[114](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:114)
Begin parsing coverage...
[115](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:115)
Parsing coverage ended
[116](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:116)
Base coverage collection ended
[117](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:117)
Begin switching back to original branch...
[118](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:118)
/usr/bin/git fetch --all --depth=1
[119](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:119)
Fetching origin
[120](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:120)
/usr/bin/git checkout -f mward-sudo/issue60
[121](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:121)
Switched to a new branch 'mward-sudo/issue60'
[122](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:122)
branch 'mward-sudo/issue60' set up to track 'origin/mward-sudo/issue60'.
[123](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:123)
Switching back to original branch ended
[124](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:124)
Begin stages.checkthreshold...
[125](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:125)
stages.checkThreshold ended
[126](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:126)
Begin generating report...
[127](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:127)
Generating report ended
[128](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:128)
Begin report publish...
[129](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:129)
Report publish ended
[130](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:130)
Begin failed tests' annotations publication...
[131](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:131)
Failed tests' annotations publication skipped
[132](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:132)
Failed tests' annotations publication ended
[133](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:133)
Begin coverage annotations publication...
[134](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:134)
Coverage annotations publication ended
[135](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:135)
Error: Jest coverage report action failed
[136](https://github.com/mward-sudo/better-switch/runs/5294543099?check_suite_focus=true#step:3:136)
Error: Process completed with exit code 1.
```

--- a/jest.config.cjs
+++ b/jest.config.cjs
@@ -3,7 +3,7 @@ module.exports = {
   preset: 'ts-jest/presets/default-esm',
   testEnvironment: 'node',
   coverageThreshold: {
-    al: {
+    global: {
       lines: 80,
     },
   },

